<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientMainWindow</class>
 <widget class="QMainWindow" name="ClientMainWindow">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client - File Transfer System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <!-- 用户身份验证区域 -->
    <item>
     <widget class="QGroupBox" name="groupBoxAuth">
      <property name="title">
       <string>User Authentication</string>
      </property>
      <layout class="QFormLayout" name="formLayoutAuth">
       <item row="0" column="0">
        <widget class="QLabel" name="labelUsername">
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditUsername"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPassword">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayoutAuth">
         <item>
          <widget class="QRadioButton" name="radioButtonRegister">
           <property name="text">
            <string>Register</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonLogin">
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSubmit">
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <!-- 分割线 -->
    <item>
     <widget class="QFrame" name="line">
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <!-- 文件传输区域 -->
    <item>
     <widget class="QGroupBox" name="groupBoxTransfer">
      <property name="title">
       <string>File Transfer</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutTransfer">
       <item>
        <widget class="QSplitter" name="splitterFiles">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="localWidget">
          <layout class="QVBoxLayout" name="verticalLayoutLocal">
           <item>
            <widget class="QLabel" name="labelLocalFiles">
             <property name="text">
              <string>Local Files</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidgetLocal"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="remoteWidget">
          <layout class="QVBoxLayout" name="verticalLayoutRemote">
           <item>
            <widget class="QLabel" name="labelRemoteFiles">
             <property name="text">
              <string>Remote Files</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidgetRemote"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutButtons">
         <item>
          <widget class="QPushButton" name="pushButtonUpload">
           <property name="text">
            <string>Upload</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonDownload">
           <property name="text">
            <string>Download</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRefresh">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowserLog">
         <property name="placeholderText">
          <string>Log Messages...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>

